petri/src/vm/construct.rs

```diff
@@ -336,7 +336,8 @@ impl PetriVmConfig {
             virtio_devices: vec![],
             #[cfg(windows)]
             vpci_resources: vec![],
-            vmgs_file: None,
+            vmgs_disk: None,
```

Comment:
Can we add a vmm_test that exercises these?

Reply:
What'd you have in mind?